docker/betanet-emulator.yml

```diff
@@ -5,6 +5,7 @@ services:
     init: true
     restart: unless-stopped
     environment:
+      RADIXDLT_HOST_IP_ADDRESS: core
```

Comment:
Most of the docker-compose files changed to include an environment variable for host IP.